src/models/document-content.ts

```diff
@@ -40,14 +40,20 @@ export const DocumentContentModel = types
         }
       }));
     },
-    addTextTile() {
+    addTextTile(initialText?: string) {
       self.tiles.push(ToolTileModel.create({
         content: {
           type: "Text",
-          text: ""
+          text: initialText
         }
       }));
     },
+    deleteTile(tileId: string) {
```

Comment:
Since models are easy to test, I would consider adding one here

Reply:
Agreed. It was late and I wanted to get it in.